raiden_contracts/constants.py

```diff
@@ -29,8 +29,11 @@
 EVENT_ADDRESS_REGISTERED = 'AddressRegistered'
 
 # Timeouts
-SETTLE_TIMEOUT_MIN = 6
-SETTLE_TIMEOUT_MAX = 2700000
+TEST_SETTLE_TIMEOUT_MIN = 5
+TEST_SETTLE_TIMEOUT_MAX = 100000
```

Comment:
Just pulled the values out of my ass, we can revert to the old ones if they have meaning.